scripts/openshiftci-e2e-4x-psi-tests.sh

```diff
@@ -32,5 +32,6 @@ echo "Getting ci-firewall, see https://github.com,/mohammedzee1000/ci-firewall"
 curl -kLO https://github.com/mohammedzee1000/ci-firewall/releases/download/$CI_FIREWALL_VERSION/ci-firewall-linux-amd64.tar.gz
 tar -xzf ci-firewall-linux-amd64.tar.gz
 
-./ci-firewall request --mainbranch main --sendqueue $SENDQUEUE --sendtopic $SENDTOPIC --sendexchange $SENDEXCHANGE --setupscript $SETUPSCRIPT  --runscript $RUNSCRIPT  --timeout 4h00m
-
+#debug purpose
+echo $JOB_NAME
+./ci-firewall request --mainbranch main --sendqueue $SENDQUEUE --sendtopic $SENDTOPIC --sendexchange $SENDEXCHANGE --setupscript $SETUPSCRIPT  --runscript $RUNSCRIPT  --timeout 4h00m --jenkinsProject $JOB_NAME
```

Comment:
I think it would be better if we wrote the correct job name explicitly tbh rather than the env as the problem we seem to be facing is with `JOB_NAME` env

Reply:
you could use different env to specify job name and and then use than env?

Reply:
@mohammedzee1000 I guess then we can remove `echo $JOB_NAME` that i added for debug purpose ?